Jenkinsfile

```diff
@@ -76,10 +81,16 @@ pipeline {
       }
     }
     stage("Tier 1") {
+      environment {
```

Comment:
You can define variable above pipeline definition so above line 16 like:
`def email_arg = ""`

and here you can use:
```groovy
script {
  if(params.EMAIL != '') {
    email_arg = "--email=${params.EMAIL}"
  }
}
```

then on line 93 you can directly use variable from groovy:
`${email_arg}`

Reply:
Thanks for the tip!